vpc.go

```diff
@@ -153,13 +154,17 @@ func (c *VPCCollector) collectVpcsPerRegionQuota(ch chan<- prometheus.Metric) {
 func (c *VPCCollector) collectVpcsPerRegionUsage(ch chan<- prometheus.Metric) {
 	ctx, cancelFunc := context.WithTimeout(context.Background(), c.e.timeout)
 	defer cancelFunc()
-	describeVpcsOutput, err := c.ec2.DescribeVpcsWithContext(ctx, &ec2.DescribeVpcsInput{})
+	var vpcs []*ec2.Vpc
+	err := c.ec2.DescribeVpcsPagesWithContext(ctx, &ec2.DescribeVpcsInput{}, func(page *ec2.DescribeVpcsOutput, lastPage bool) bool {
+		vpcs = append(vpcs, page.Vpcs...)
```

Comment:
Also here, i'd suggest to only save the actual number of vpcs, and not the vpcs to make it clearer what we are attempting to do.